### PR TITLE
Docs Restructure (Part 1)

### PR DESCRIPTION
**Pull request by sofinico**
## Description 

Documentation cleaning 🧹. This is the first phase of restructuring; the final version will follow in the next few days.

Impacts on https://docs.alexlab.co/. This branch is live at a temporary URL [here](https://coinfabrik.gitbook.io/alexlab-doc/restructure-1). 

1. A `./legacy/` directory was created for outdated files that are no longer live. Assets that were outdated and unused have been removed from the repository.

2. Emojis were added to Features subpages. The staking feature layout has been applied, with content updates planned in a future specific PR.

4. Pages: Official Links, Security Audits, Whitepapers and Token List were grouped under "Resources" title.

5. Tokenomics page was moved to the "Detailed Information" group. 

**Note:** No content updates were performed. This is purely a refactor PR.

## Dev notes

Please merge #5 first. The changes relative to this PR are from https://github.com/alexgo-io/alexlab-doc/pull/6/commits/f2a26bb07e518b928d5fd5edf10872c2477eb517 ahead.

### CODE REVIEW

**Review by tenuki — APPROVED** [review]
So far so good !

File: product-features/farming/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    visible: true
+  pagination:
+    visible: false
+---
+
+# 🌾 Farming
+
+Yield Farming on ALEX is an excellent way to earn exciting returns. By farming (staking or locking up your LP tokens), you earn rewards in addition to the earnings from being a liquidity provider. Among these rewards are $ALEX and APower tokens.
+
+To get started with farming, you first deposit two tokens into a liquidity pool, receiving LP tokens in return. Then, you stake these LP tokens in a farm, allowing you to harvest rewards after each staking cycle while still benefiting from the trading fees generated by the pool.
```
Comment (oscartsui) [inline product-features/farming/README.md:20]: should be each farming cycle instead of staking cycle

File: product-features/farming/faqs.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+<details>
+
+<summary>What is APower?</summary>
+
+ALEX Staking Power, or APower, is a non-transferrable and non-tradable token. It is a special incentive that you can earn through staking on the ALEX Lab Platform, either by:
+
+1. **Stake $ALEX (1x Multiplier)**
+2. **Stake LP tokens through Yield Farming (0.3x Multiplier)**
+
+APower is the lottery ticket that allows you to take part in any future IDO rounds on our [Launchpad](https://app.alexlab.co/launchpad). There is no maximum amount of APower an address can earn over a period of time. If you are interested in frequently participating in IDOs, staking $ALEX would generate APower fastest.
```
Comment (oscartsui) [inline product-features/farming/faqs.md:86]: I think I have changed the wording of APower somewhere in either pull4 or pull5. Can you make it the same to keep it consistent?

File: product-features/farming/how-to.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+For example, you can manually stake your \$ALEX rewards to generate compounding interest. If your rewards are another token, you can still [swap](https://app.alexlab.co/swap) and convert them to \$ALEX. Another option would be to use your rewards to buy more LP tokens and benefit from being a [liquidity provider](../liquidity-providers/README.md).
+
+Happy Farming! 🥕 🥬 🍅
+
+## :bellhop: Withdrawing LP tokens (unstake)
+
+Withdrawing you LP tokens takes just a few steps. The important thing is when to do it.
+
+When farming, you are commit to locking up your tokens for a predefined period (reward cycles, each one ~3.5 days). Once these cycles conclude, you will be able to unstake them and regain control over your LP tokens.
```
Comment (oscartsui) [inline product-features/farming/how-to.md:192]: When farming, you are committed to locking up your tokens for a predefined period (reward cycles, each one ~3.5 days). Once these cycles conclude, you will be able to unstake them and regain control over your LP tokens.

File: product-features/farming/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: false
+---
+
+# 🌾 Farming
+
+Yield Farming on ALEX is an excellent way to earn exciting returns. By farming (staking or locking up your LP tokens), you earn rewards in addition to the earnings from being a liquidity provider. Among these rewards are $ALEX and APower tokens.
```
Comment (Hadan-ALEX) [inline product-features/farming/README.md:18]: Yield Farming on ALEX is an excellent way to earn exciting returns. By farming (staking or locking up your LP tokens) you earn rewards in addition to the pool fees from being a liquidity provider. Among these rewards are $ALEX and APower tokens.

File: product-features/farming/faqs.md (hunk 2 above)
Comment (Hadan-ALEX) [reply product-features/farming/faqs.md:86]: This is what we used for PR#5:

APower is the access token for participating in the "Community Round" of any future IDO on our [Launchpad](https://app.alexlab.co/launchpad). IDO tickets allocated to this round can only be purchased by utilizing APower. There is no maximum amount of APower an address can earn over a period of time. If you are interested in participating in IDOs, staking $ALEX would generate APower fastest.

File: product-features/farming/key-concepts.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+description: >-
+  All you need to know for successfully farming on ALEX Lab, from farm basics to
+  dashboard metrics!
+---
+
+# Key concepts
+
+## Farm basics
+
+Yield farming works very similar to standard staking, with the key difference being that the tokens you stake are LP tokens. As in traditional staking, you lock up your tokens for a certain period (measured in cycles) and earn rewards over time. After each cycle, you will have rewards available to harvest.
```
Comment (Hadan-ALEX) [inline product-features/farming/key-concepts.md:11]: Yield farming works in a very similar way to standard staking, with the key.....

File: product-features/farming/faqs.md (hunk 2 above)
Comment (sofinico) [reply product-features/farming/faqs.md:86]: Yes! This was already changed in previous PR, so changes will be applied when merging.

**Comment by sofinico** [comment]
Ready to merge! 🚀

**Pull request merged**